port/unix_include/omrportpg.h

```diff
@@ -120,7 +121,8 @@ typedef struct OMRPortPlatformGlobals {
 #endif
 
 #if defined(LINUX)
-#define PPG_cgroupLimitsEnabled (portLibrary->portGlobals->platformGlobals.cgroupLimitsEnabled)
+#define PPG_cgroupSubsystemsAvailable (portLibrary->portGlobals->platformGlobals.cgroupSubsystemsAvailable)
```

Comment:
There are three port library globals in play here and it looks like the two new ones are only valid when `PPG_cgroupEntryList ` is non-null?

Can you add a comment here (and in the commit) that describes the relationship between the three port globals?

Reply:
Can you add the above invariant as a comment in the code?

Reply:
Done